libnmstate/nm/connection.py

```diff
@@ -113,8 +113,20 @@ def create_new_nm_simple_conn(iface, nm_profile):
     if nm_profile and not is_multiconnect_profile(nm_profile):
         con_setting.import_by_profile(nm_profile, iface.is_controller)
     else:
+        # OVS bridge and interfaces could sharing the same interface name, to
+        # distinguish them at NM connection level, instead of using interface
+        # name as connection name, we append a prefix
```

Comment:
Actually, your code is adding a postfix. :)

Reply:
I changed all the references to "postfix", sorry about that. Thanks!